Detectors/CTP/macro/TestConfig.C

```diff
@@ -16,13 +16,13 @@
 #endif
```

Comment:
@lietava could you add 
```
#include "Framework/Logger.h"
```
after(!) the the `#endif` to pass macro test in the interpreter mode?

Reply:
@lietava the `#include "Framework/Logger.h"` should have been added *after* the `#endif`, to be visible in the interpreter mode, which is currently failing.